docs/content/preview/drivers-orms/csharp/postgres-npgsql.md

```diff
@@ -90,14 +87,16 @@ connStringBuilder.TrustServerCertificate = true;
 CRUD(connStringBuilder.ConnectionString);
 ```
 
-| Parameter | Description | Default |
+| Npgsql Parameter | Description | Default |
 | :---------- | :---------- | :------ |
 | sslmode  | SSL Mode | require
 | TrustServerCertificate |  Trust the server certificate configured on the YugabyteDB cluster | true
 
+If you created a cluster on [YugabyteDB Managed](https://www.yugabyte.com/managed/), use the cluster credentials and [download the SSL Root certificate](/preview/yugabyte-cloud/cloud-connect/connect-applications/). Refer to [Use C# with SSL](../../../quick-start/build-apps/csharp/ysql/#use-c-with-ssl) for the default and supported modes.
```

Comment:
Except Use C# with SSL says disable is supported - which it is by the driver - but is not by YB Managed...

Also, on the Use C# with SSL page there is a double colon typo